### ISSUE TITLE
Browser refresh while logged in forces reauthentication

### ISSUE THREAD

**Issue by aajacobs** [marklogic-community/slush-marklogic-node#522]
I created a new app using 1.3.2 of the generator, and each time I refresh the browser I am taken back to the login screen and have to reauthenticate.  So I created another app on a different MarkLogic instance just to be sure and still had the problem.

**Comment by grtjn** [comment]
I have been observing this too, particularly with ml 9.0-3. Thanks for reporting, i will make sure it gets investigated..
-------- Oorspronkelijk bericht --------Van: aajacobs <notifications@github.com> Datum: 25-10-17  16:22  (GMT+01:00) Aan: marklogic-community/slush-marklogic-node <slush-marklogic-node@noreply.github.com> Cc: Subscribed <subscribed@noreply.github.com> Onderwerp: [marklogic-community/slush-marklogic-node] Browser refresh while
  logged in forces reauthentication (#522) 
I created a new app using 1.3.2 of the generator, and each time I refresh the browser I am taken back to the login screen and have to reauthenticate.  So I created another app on a different MarkLogic instance just to be sure and still had the problem.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/marklogic-community/slush-marklogic-node","title":"marklogic-community/slush-marklogic-node","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/marklogic-community/slush-marklogic-node"}},"updates":{"snippets":[{"icon":"DESCRIPTION","message":"Browser refresh while logged in forces reauthentication (#522)"}],"action":{"name":"View Issue","url":"https://github.com/marklogic-community/slush-marklogic-node/issues/522"}}}
